### DISCUSSION TITLE
SVT Play - Missing text file

### DISCUSSION THREAD

**Discussion by Sniggelurman** [stefansundin/privatkopiera#233] [General]
Dear All,
In most SVTPlay videos, there is a 'Text open' stream that can be downloded as .VTT. But what to do when there is no such stream? (See example attached).

Thanks in advance!

![20241104_114737](https://github.com/user-attachments/assets/c50a84c6-6a1a-42d0-8444-baf05b614879)

**Comment by Lunarwalker60** [comment]
Om det är ett program med utländskt tal och det inte finns någon .VTT-fil så är texten mest troligt inbränd i videofilen.

**Comment by Sniggelurman** [comment]
Tack för snabbt svar, Stefan! Ja, texterna kan vara inbrända men i de fallen så slutar ljudet att fungera en bit in i programmet. Noterar att i dessa fall (som bilden jag bifogade) finns det en separat audioström, men har inte lyckats tillgodogöra mig den.

Antar att det är någon form av kopieringsskydd, för det uppträder mest när det gäller nyare produktioner.

**Comment by Sniggelurman** [comment]
Hej igen Stefan,
Detta är svar nr 2 på ditt mail. Kollade nu med att ladda ner avsnitt 1 av Das Boot säsong 4 (Das Boot S04E01) https://www.svtplay.se/video/KZm2vvw/das-boot/avsnitt-1?info=visa  Texterna är inbrända, men å andra sidan så slutar ljudet fungera efter 3 min 27 sek. Har också noterat att det kan vara problem även med Chromecast för dessa filer. Jag kan inte tekniken i detta, men det förefaller som det har med kopieringsskydd att göra. Har testat olika uppspelningsprogram (inkl VLC som brukar klara det mesta), men med samma resultat.

Likartade resultat med alla SVT Play-filer som inte erbjuder en "Text-open-ström".

Hoppas detta kan fånga ditt intresse!

**Comment by Lunarwalker60** [comment]
För ordnings skull påpekar jag att det inte är jag som är Stefan. Jag har varit engagerad länge i forumet och lärt mig mycket av Stefan, prenumererar på "issues" och försöker förmedla vad lärt mig här. Jag har laddat hem avsnittet som Du har problem med och det fungerar utan problem hos mig. Text OK, ljudet OK hela tiden både med GOM-player på Win 10 PC och med Kodi player. Jag kan tyvärr inte hjälpa dig med lösningen på detta.

**Comment by Sniggelurman** [comment]
Hej Lunarwalker,
Ber om ursäkt för att jag felaktigt kallade dig för Stefan, men upplevde att du var så initierad i frågeställningarna så jag antog att du var han...
Jag är medveten om att det gått väldigt lång tid sedan vi hade kontakt senast, men nu under julhelgerna har jag haft lite tid att titta vidare på problemet med ljudet som försvinner vid "inbrända" texter på vissa filer på SVTPlay. Det visar sig att det är helt konsekvent så att ljudet försvinner efter exakt 3 min 27 sek. Har även testat några gånger med att ladda ner bara ljudfilen (audio-strömmen som Privatkopiera erbjuder) och noterar att dessa filer också alltid blir just 3 min och 27 sekunder långa. Så det finns uppenbarligen någon slags systematik här... Noterar också att denna problematik i första hand verkar förekomma i amerikanska filmer, men delvis också i engelska, tyska och franska. Jag funderar på om det vara så att amerikanska (och vissa engelska etc rättighetsinnehavare) ställer högre krav på kopieringsskydd? Eller varför använder SVTPlay annars olika sätt att hantera undertexter (dvs varför finns det inte alltid en "text-open.vtt"-fil tillgänglig?).

Ursäkta att jag är så envis med mina frågor, men det är väldigt frustrerande att jag inte lyckas få ordning på detta. Jag använder Win10 och Firefox.

God fortsättning!

**Comment by Lunarwalker60** [comment]
@Sniggelurman - det enda jag kan rekommendera dig är att prova med en annan version av ffmpeg. Det är tyvärr inte så att bara för att en utgåva är nyare att den autmatiskt är bättre på allt. Det finns så många grenar i utvecklingen och den Du använder kanske inte härstammar från samma gren som mina. Jag har tex olika versioner för secure och öppna nerladdningar. Jag har tyvärr inte möjlighet att ge dig några exakta build-versioner just nu - chansa och hugg en som ser bra ut och testa tills Du hittar en som fungerar.

**Comment by Sniggelurman** [comment]
OK, stort tack för supersnabbt svar. Jag ska se till att "labba" lite med olika versioner. Men skulle du så småningom få möjlighet att kolla vilken version du har så skulle jag självfallet vara väldigt tacksam!

**Reply by Lunarwalker60** [reply]
Nu har jag testat att ladda ner Das Boot S04E01 och det gick bra med text och ljud. Jag använder en gammal version av ffmpeg för secure-videos. Jag vet inte var Du skall hitta den om den ens finns att finna någonstans längre. Men som sagt; det finns säkert nyare versioner som också fungerar men jag har inte sökt eftersom den jag använder fungerar.

ffmpeg version 4.2.2 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 9.2.1 (GCC) 20200122
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libdav1d --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --enable-libmfx --enable-amf --enable-ffnvcodec --enable-cuvid --enable-d3d11va --enable-nvenc --enable-nvdec --enable-dxva2 --enable-avisynth --enable-libopenmpt
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100

**Comment by webberian** [comment]
Jag har `ffmpeg version 5.1.6-0+deb12u1 Copyright (c) 2000-2024` på min Linux-dator, och kan inte heller återskapa problemet med att ljudet försvinner.
